.github/workflows/release.yml

```diff
@@ -35,6 +35,30 @@ jobs:
 
           echo ::set-output name=version::$version
 
+      - name: Import GPG Key For Tags
+        id: tag_gpg
+        uses: hashicorp/ghaction-import-gpg@78437f97569a473e42b227be84d4084c2dfb49ba # v2.1.0
+        env:
+          GPG_PRIVATE_KEY: ${{ secrets.GPG_PRIVATE_KEY }}
+          PASSPHRASE: ${{ secrets.GPG_PRIVATE_KEY_PASSWORD }}
+
+      - name: Import GPG Key For Terraform
+        id: import_gpg
+        uses: hashicorp/ghaction-import-gpg@78437f97569a473e42b227be84d4084c2dfb49ba # v2.1.0
+        env:
+          GPG_PRIVATE_KEY: ${{ secrets.RELEASE_GPG_PRIVATE_KEY }}
+          PASSPHRASE: ${{ secrets.RELEASE_GPG_PRIVATE_KEY_PASSWORD }}
```

Comment:
We now use two GPG keys, one for `signing of tags and commits` and the other for `Terraform`. They are to be in environment named `release`